deploy/kamaji-azure.env

```diff
@@ -12,10 +12,10 @@ export KAMAJI_SUBNET_ADDRESS=10.224.0.0/16
 export KAMAJI_NAMESPACE=kamaji-system
 
 # tenant cluster parameters
-export TENANT_NAMESPACE=default
+export TENANT_NAMESPACE=kamaji-system
```

Comment:
Why deploying a Tenant Control Plane in the same Kamaji Namespace?

Reply:
just typo